client/src/containers/AddTrip/__tests__/index.spec.js

```diff
@@ -82,7 +141,7 @@ describe('<AddTrip />', () => {
     it('should dispatch addTrip when submitForm is called', () => {
         const preventDefault = jest.fn();
         const props = {
-            user: {
+            users: {
```

Comment:
Even 100% test coverage did not save me from this bug that was breaking creation of the review.